CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Change Log
 
+## [0.9.0](https://github.com/redhat-developer/vscode-xml/milestone/10?closed=1) (September 10, 2019)
+
+### Enhancements
+
+ * Add support for `textDocument/documentHighlight` for DTD. See[#545](https://github.com/angelozerr/lsp4xml/issues/545)
```

Comment:
all lsp4xml issues should be displayed as `[lsp4xml#issue](link)`, to hint at the link to another repo